src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,111 @@
+    github: canyon289
+  - name: Oriol Abril-Pla
+    github: OriolAbril
+  - name: Osvaldo Martin
+    github: aloctavodia
+summary: "Xarray provides the core data model for ArviZ's InfrerenceData structure"
+---
+
+_TLDR: Xarray is the core of the `az.InferenceData` object. [ArviZ InferenceData](https://python.arviz.org/en/stable/api/generated/arviz.InferenceData.html) simplifies the Bayesian workflow, facilitates reproducibility, and enables interoperability between different Probabilistic Programming languages._
+
+<!-- Temporary Reference https://xarray.dev/blog/introducing-pint-xarray -->
+
+```python
+# Load the saved results of a bayesian analysis from disk
+import arviz as az
+
+data = az.load_arviz_data("centered_eight")
+data
+```
+
+<!-- https://xarray.dev/blog/introducing-pint-xarray -->
+
+## The Life of a modern Bayesian
+
+The utility of `az.InferenceData` will not be very apparent if we first don't talk about the life of a Modern Bayesian Statistician,
+the PPL Designer, and what things used to be like before `az.InferenceData`.
+
+### The Modern Bayesian Practitioner
+
+Like most modern statisticians, the Modern Bayesian Statistician uses a computer to perform their work.
+Many Probabilistic Programming Language (PPL) are available for this task, such as [Stan](https://mc-stan.org/),
+[PyMC](https://www.pymc.io/), [Tensorflow Probability](https://www.tensorflow.org/probability), [NumPyro](http://pyro.ai/numpyro/) etc.
+
+With probabilistic programming language in hand, the Bayesian then follows the "Bayesian workflow" below, performing each of the various steps
+as part of their analysis.
+
+![Bayesian Workflow](https://bayesiancomputationbook.com/_images/Bayesian_workflow.png)
+
+At each of these steps the PPLs generate many arrays, and not only single dimension arrays, but multi dimensional arrays.
+The prior predictive distribution, the posterior distribution, the posterior predictive distribution, the log likelihood, and even the input data
+are all numerical quantities that need to be captured and ordered to facilitate analysis.
+
+Often these arrays need to be passed to specialized functions in order to check for MCMC convergence, model comparison, or getting summaries from the target distribution.
+
+And once the individual statistician has completed their work, they may either want to save it to disk,
+or share it with a colleague.
+
+## The Modern PPL designer
+
+Now consider a PPL designer. PPL designers typically want to focus on model building and inference: graph representation, samplers, symbolic differentiation speed and stability...
+However for their PPL to be useful they must ensure users have easy access to diagnostics,
+visualizations, and model criticism tools.
+Suddenly the scope of their codebase grows, and duplicative functionality now exists across
+the open source Bayesian ecosystem
+
+## Life before az.InferenceData
+
+Before `az.InferenceData` there wasn't a consistent object or method
```

Comment:
Can you add a line on what "object" and "method" represent please

Reply:
Yes

Reply:
Added a bit more explanation here.